src/term.rs

```diff
@@ -836,17 +891,34 @@ pub enum UnaryOp {
     /// Transform a string to an enum.
     EnumFromStr(),
     /// Test if a regex matches a string.
-    /// Like [`StrMatch`], this is a unary operator because we would like a way to share the
+    /// Like [`UnaryOp::StrMatch`], this is a unary operator because we would like a way to share the
```

Comment:
Just out of curiosity, didn't `` [`StrMatch`] `` work? I thought we tested those links when building the documentation of 0.1.0.

Reply:
No, they're only highlighted as code. I discovered this while adding documentation for `Force` and testing how it renders.

Just doubled-checked on master.